### ISSUE TITLE
PDF417 codes does not work

### ISSUE THREAD

**Issue by omerbhutta** [mebjas/html5-qrcode#499]
**I am trying to integrate HTML  bar code version and you mention in the documentation that the script has functionality to scan PDF417 but it didn't.**

=====================
Code that i am using

function docReady(fn) {
    if (document.readyState === "complete"
            || document.readyState === "interactive") {
        setTimeout(fn, 1);
    } else {
        document.addEventListener("DOMContentLoaded", fn);
    }
}

        docReady(function () {
            var resultContainer = document.getElementById('qr-reader-results');
            var lastResult, countResults = 0;
            function onScanSuccess(decodedText, decodedResult) {
                if (decodedText !== lastResult) {
                    ++countResults;
                    lastResult = decodedText;
                    console.log(`Scan result ${decodedText}`, decodedResult);
                }
            }

            const formatsToSupport = [Html5QrcodeSupportedFormats.PDF_417];

            const html5QrcodeScanner = new Html5QrcodeScanner(
                    "qr-reader",
                    {
                        fps: 10,
                        qrbox: {width: 250, height: 250},
                        formatsToSupport: formatsToSupport
                    },
                    /* verbose= */ false);
            html5QrcodeScanner.render(onScanSuccess);
        });

        function onScanSuccess(decodedText, decodedResult) {
            console.log(`Code matched = ${decodedText}`, decodedResult);
        }
====================

**Comment by ROBERT-MCDOWELL** [comment]
OS and browser version, error message if any

**Comment by omerbhutta** [comment]
It only shows that QR code format is not supported.

**Comment by rlyle** [comment]
Same issue here too.. unable to scan PDF_417 in chrome Version 103.0.5060.114 (Official Build) (64-bit)

**Comment by ROBERT-MCDOWELL** [comment]
change to experimentalFeatures: { useBarCodeDetectorIfSupported: true}

**Comment by rlyle** [comment]
I tried it both true and false, no change.

**Comment by ROBERT-MCDOWELL** [comment]
useBarCodeDetectorIfSupported to true means the standard javascript API engine is taking the relay.
if this is not working, so it surely comes from the barcode you are scanning. is the same barcode you are using for test works with other online scanner?

**Comment by rlyle** [comment]
Yes, just testing with my own DL. The BlinkID PDF_417 scanner worked, but their pricing for a license was through the roof. I noticed this project uses the ZXing library, which I've also tested and it didn't work either. In fact, their own online demo doesn't even manage to scan their test images for PDF_417 either.  (see https://zxing-js.github.io/library/examples/pdf417-image/)

So this might be a bug in the underlying ZXing library.

**Comment by gabrielsolomon** [comment]
I have noticed that for PDF417 orientation matters. 
It seems to detect them only in certain orientation (horizontal)

**Comment by chupetoide** [comment]
same issue here testing with Chrome Version 110.0.5481.177 (Official Build) (x86_64) on MacOS.

**Comment by pkasidit** [comment]
I have the same issue with Chrome Version 120.0.6099.129 (Official Build) (arm64) on MacOS.
